models/Repository.ts

```diff
@@ -13,18 +13,10 @@ export interface GitRepository extends Repository {
 export type Organization = components['schemas']['organization-full'];
 
 const getGitIssues = memoize(async (URI: string) => {
-  const { body: issuesList } = await githubClient.get<Record<string, number>>(
+  const { body: issuesList } = await githubClient.get<Record<string, any>>(
     `repos/${URI}/issues`,
   );
-  console.log('issueCount====', issuesList);
-  //   const issueAverage = averageOf(...Object.values(issueCount!));
-
-  //   const issuesList = Object.entries(issueCount!)
-  //     .filter(([_, score]) => score >= issueAverage)
-  //     .sort(([_, a], [__, b]) => b - a);
-
-  //   return issuesList.map(([name]) => name);
-  return issuesList;
+  return issuesList!.filter((issue: any) => !issue.pull_request);
```

Comment:
```suggestion
  return issuesList!.filter(({ pull_request }) => !pull_request);
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#unpacking_properties_from_objects_passed_as_a_function_parameter